it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -319,14 +326,16 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
             journeyConfig = testSoleTraderJourneyConfig
           ))
           stubAuth(OK, successfulAuthResponse())
-          stubRetrieveIndividualDetails(testJourneyId)(OK, testIndividualDetailsJsonNoSautr)
+          stubRetrieveIndividualDetails(testJourneyId)(OK, testIndividualDetailsJsonNoSautr).setNewScenarioState("auditing")
```

Comment:
I think we need to be careful here. The SoleTraderIdentificationService method "retrieveIndividualDetails" is invoked by the Submission service and the audit service, but in the latter case only if the journey is an individual journey. In this case we are following the sole trader journey and as such only have to stub a single invocation of "retrieveIndividualDetails".

Please investigate and, if I am right, update the tests as necessary.

Reply:
This is the test you were talking about .... I added only a fake call to    stubRetrieveES20Result(testJourneyId)(NOT_FOUND) because otherwise wiremock complains that call has not been mocked.
<img width="777" alt="Screenshot 2022-04-05 at 09 19 41" src="https://user-images.githubusercontent.com/4958044/161711532-e90cc3da-6326-43e5-8b6b-1cf012f0530f.png">